src/utils/index.ts

```diff
@@ -27,7 +27,7 @@ export const isValidNumber = <T,>(value: number | string | T): boolean =>
  *
  */
 export const isEmptyList = <T,>(obj: Array<T> | T): boolean =>
-    !Array.isArray(obj) || (Array.isArray(obj) && obj.length === 0);
+    Array.isArray(obj) && obj.length === 0;
```

Comment:
Was the earlier implementation causing issues on some values?

Reply:
I didnt understand well the meaning of function, and you explained below on comments. I'll fix it,